README.rst

```diff
@@ -84,6 +84,13 @@ Synapse Installation
 For details on how to install synapse, see `<INSTALL.md>`_.
 
 
+To actually run your new homeserver, pick a working directory for Synapse to
```

Comment:
this is only correct if you installed from source; I moved it out deliberately

Reply:
kk. It's also in INSTALL.md, should I leave it there?